docs/persistenceClustering1.md

```diff
@@ -0,0 +1,62 @@
+# Persistence Clustering 1 Example
+
+<!--[![Dragon example video tutorial](https://topology-tool-kit.github.io/img/gallery/dragon.jpg)](https://youtu.be/YVk9vRKIEX8)-->
+
+<!--iframe width="100%" height="420"
+src="https://www.youtube.com/embed/YVk9vRKIEX8" frameborder="0"
+allowfullscreen></iframe-->
+
+![Persistence Clustering 1 Image](https://topology-tool-kit.github.io/img/gallery/persistenceClustering1.jpeg)
+
+## Pipeline description
+<!--This example first loads a triangle mesh from disk.-->
+First, this example loads a point cloud from disk (top left view in the above screenshot), then it computes a mesh on which a density field is obtained with a Gaussian Resampling on the points (top right view in the above screenshot). This density field will be considered as the input scalar data.
```

Comment:
I'd suggest to mention the origin of the data set (see https://topology-tool-kit.github.io/examples/persistenceClustering0/)